packages/now-cli/src/util/dev/server.ts

```diff
@@ -182,6 +182,20 @@ export default class DevServer {
     const filesChanged: Set<string> = new Set();
     const filesRemoved: Set<string> = new Set();
 
+    const distPaths: string[] = [];
+
+    for (const buildMatch of this.buildMatches) {
+      for (const buildResult of buildMatch[1].buildResults) {
```

Comment:
Why is `buildMatch[1]` hard coded to the second build match?

Reply:
@styfle because `buildMatches` is a map, so `buildMatch[0]` is the key, and we just need the value

Reply:
Made it use `.values()`, so it's more obvious